docs/_includes/sum-toc.adoc

```diff
@@ -4,33 +4,41 @@ Included in:
 - user-manual
 ////
 
-[cols="1m,1,2m,2,1"]
+[cols="1,1,2,2a,1"]
 .Table of contents attributes and values
 |===
 |Attribute |Values |Example Syntax |Comments |Backends
 
-.3+|toc
-.3+|auto, left, right, macro, preamble
-.3+|:toc:
-
+.4+|toc
+|auto, left, right, macro, preamble
+|`:toc: left`
 |Defaults to `auto` if value is blank.
 |html
 
-|_not set_, `macro`, and `preamble` only
+|macro, preamble
```

Comment:
Since 1.5.3, the auto location now works with embeddable HTML. I'll go ahead and add that in.